### ISSUE TITLE
Use default extension path if it's available

### ISSUE THREAD

**Issue by petebankhead** [qupath/qupath#1260]
## Feature request

**Is your feature request related to a problem? Please describe.**
QuPath currently requires the extension path to be set explicitly - even if the default path already exists. This can sometimes be annoying.

See https://forum.image.sc/t/default-extension-location-unset-by-default-problem-for-pre-installing-extensions/79594

**Describe the solution you'd like**

Proposed solution:

On startup, QuPath should check if there is an extension path set. If there is not, then it should check if a directory corresponding to the default path exists.
* If yes, ask the user whether to use this path
* If no, continue silently

Note that if the user does not want to use the path, there should be an option to select another path or to not ask again - otherwise the prompt will appear every time the software is started.

**Describe alternatives you've considered**
1. The default path could always be used if the directory exists, even if the path hasn't been specified.
    * This could be troublesome, because it means that extensions might be installed unintentionally. In the worst cases, if an extension is causing the problems that resulted in the QuPath reinstallation, then startup might fail.
2. There could be an option to set the extension path (including to the default) from the command line, which could be applied post-install.
   * This is already somewhat possible with a one-line script, see https://forum.image.sc/t/qupath-extension-folder-on-distant-server/78190/2?u=petebankhead

**Additional context**
An idea from the forum post was to set the extension path during installation, but I don't think there's a way to modify the installer sufficiently to get that to work.

**Comment by petebankhead** [comment]
Closing this as I don't think it's an issue since the extension manager was added.

**Comment by alanocallaghan** [comment]
I agree it's no longer an issue, but I hope we're not doing what's described in the case that the option isn't set and the directory doesn't exist. Adding an extra click to confirm the use of the default extension directory either way is better imo
